src/ol/source/Vector.js

```diff
@@ -275,7 +280,7 @@ class VectorSource extends Source {
     /**
      * A lookup of features by id (the return from feature.getId()).
      * @private
-     * @type {!Object<string, FeatureClass|Array<RenderFeature>>}
+     * @type {!Object<string, import('../Feature.js').FeatureLike|Array<import('../Feature.js').FeatureLike>>}
```

Comment:
I'm just curious why you didn't use `FeatureClassOrArrayOfRenderFeatures<FeatureClass>` here instead?

Reply:
I had tried, but TypeScript didn't like it.